src/Http/Controllers/CP/Collections/EntriesController.php

```diff
@@ -72,7 +72,7 @@ protected function indexQuery($collection)
             $query->where('title', 'like', '%'.$search.'%');
         }
 
-        return $query;
+        return $collection->alwaysUseSearchIndex() ? $collection->searchIndex()->ensureExists()->query() : $query;
```

Comment:
Don't bother changing this yet, because of the general comment on this PR, but...

This should go further up, to line 65 where you initially set `$query`. Otherwise, if you don't have a search index, the `->where('title', 'like', '%'.$search.'%');` gets discarded since you're overriding the query here.

Reply:
If you have search index it defers to the search() method on it in line 70
The `->where('title', 'like', '%'.$search.'%')` bit is only when you dont have a search index (?)